### PR TITLE
make deploy velero namespace fix

### PR DESCRIPTION
**Pull request by kaovilai**

### CODE REVIEW

**Review by savitharaghunathan — COMMENTED** [review]
/lgtm
Thanks, @kaovilai

File: Makefile [hunk 1]
```diff
@@ -138,17 +138,24 @@ install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~
 uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config.
 	$(KUSTOMIZE) build config/crd | kubectl delete -f -
 
-deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
+DEPLOY_TMP=/tmp/oadp-make-deploy
+deploy-tmp: kustomize
```
Comment (dymurray) [inline Makefile:142]: Does this need to clean up the tmp dir after it runs?

Comment (kaovilai) [reply Makefile:142]: Sure thing.

Comment (kaovilai) [reply Makefile:142]: It can.. but only in its dependent make targets.

Comment (kaovilai) [reply Makefile:142]: resolved in https://github.com/openshift/oadp-operator/pull/506/commits/d38c98934ec7f6402e5bb903fb24511621c19830

**Review by dymurray — APPROVED** [review]

**Pull request merged**